english/friendly-person/friendly-person.md

```diff
@@ -1,14 +1,23 @@
 As you get closer, you suddenly realize that the person on the path is you!
 You look different somehow. You look good, happy, peaceful.
 
-You ask yourself a question:
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.